.travis.yml

```diff
@@ -6,16 +6,19 @@ php:
 
 sudo: false
 
+env:
+  global:
+    - TZ=Europe/Paris
+
 cache:
   directories:
     - "$HOME/.composer/cache"
 
-before_script:
+install:
```

Comment:
See https://docs.travis-ci.com/user/customizing-the-build#The-Build-Lifecycle.

Reply:
Since we install dependencies here, it makes more sense to use the `install` section rather than `before_script`. 

There's another reason I will get to in another PR, hehe! 🤓

Reply:
:+1: